src/news/2012-10-24-jutta-zoomer-mag.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Jutta Treviranus Listed Among Zoomer Magazine’s Top 45 Over 45
+filename: jutta-zoomer-mag
+date: '2012-10-24'
+---
+Jutta Treviranus, Director of the
+[Inclusive Design Research Centre](http://idrc.ocadu.ca), has been named by
+[Zoomer Magazine](http://www.zoomermag.com/) as one of
+[Canada's Top 45 Over 45](http://www.everythingzoomer.com/arts-entertainment/stars-royals/2012/10/05/the-3rd-annual-zoomer-list-canadas-top-45-over-45-jutta-treviranus/)
```

Comment:
Please add a comma after this link. If the linter complains about it, you can add an exception in this case.